### PR TITLE
Add Windows makefile + update readme

### PR DESCRIPTION
**Pull request by ssebs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,24 +23,31 @@
...
-```
-
-2. Rock out!
-
-```bash
-. ./venv/bin/activate
-python main.py [songPath]
-```
+- Install Python3.6+
+- Mac/Linux
```
Comment (amorriscode) [inline README.md]: Can we separate the Mac/Linux instructions into sections? So we could have:

```
## Getting Started

### Mac/Linux

1. Install Python 3.6+ (link to some instructions)
2. ...

### Windows

1. Install Python 3.6+ (link to some instructions)
2. ...
```

**Review by amorriscode — COMMENTED** [review]
Thanks for your contribution @ssebs! I know it's early days but this project is going to be awesome once it's playable. 😆 

Just a minor request to update some formatting.

**Comment by ssebs** [comment]
@amorriscode How's this?

**Comment by amorriscode** [comment]
@ssebs Looks good! Do you mind rebasing with the `libogg` requirement? After that I can approve and merge it in.

**Comment by ssebs** [comment]
@amorriscode Done

**Review by amorriscode — APPROVED** [review]
LGTM! 🔥

**Pull request merged**